docs/content/faq.md

```diff
@@ -22,6 +22,20 @@ See the [remote setup docs](/remote_setup/) for more info.
 
 This has now been documented in its own [remote setup page](/remote_setup/).
 
+### How can I get rid of the "Config file not found" notice?
+
+If you see a notice like 'NOTICE: Config file "rclone.conf" not found', this
+means you have not configured any remotes.
+
+If you need to create a remote, see the [remote setup page](/remote_setup/).
+
+If you are using rclone without remotes, you can create an empty config file
```

Comment:
Strictly speaking remotes configured on the fly on the command line are still remotes.

So perhaps something like this might be better:

If you are using rclone entirely with [on the fly remotes](https://rclone.org/docs/#backend-path-to-dir), you can create an empty config file

Reply:
Fixed this as well. Thanks!